src/EndpointAndManager.sol

```diff
@@ -42,7 +43,7 @@ abstract contract EndpointAndManager is Endpoint, EndpointManager {
 
     // override this function to avoid a storage lookup. The endpoint is always enabled
     function _getEnabledEndpointsBitmap() internal pure override returns (uint64) {
-        return 0x1;
+        return uint64(1 << ENDPOINT_INDEX);
```

Comment:
Can you explain the logic behind this change pls?

Reply:
technically it's identical to before (since `ENDPOINT_INDEX` is 0) but now more consistent with the use of that constant. i.e. if we change `ENDPOINT_INDEX` to 200, it will calculate the bit correctly still